app/views/contacts/_contact_details.html.erb

```diff
@@ -22,6 +22,26 @@
   </div>
 <% end %>
 
+<% if @contact.show_webchat? %>
+  <h2 class="details-header">Web chat</h2>
+  <div class="details js-webchat">
+    <div class="description govspeak">
+      <p class="js-webchat-advisers-unavailable hidden">
+        Currently there are no advisers available to service your request, please try again later.
+      </p>
+      <p class="js-webchat-advisers-busy hidden">
+        All our advisers are currently busy, please try again later.
+      </p>
+      <p class="js-webchat-advisers-available">
```

Comment:
Does this mean with JavaScript turned off users will see this message when the chat has a dependency on JS?